pkgs/applications/misc/st/default.nix

```diff
@@ -14,7 +14,7 @@ stdenv.mkDerivation rec {
   inherit patches;
 
   configFile = optionalString (conf!=null) (writeText "config.def.h" conf);
-  preBuild = optionalString (conf!=null) "cp ${configFile} config.def.h";
+  prePatch = optionalString (conf!=null) "cp ${configFile} config.def.h";
```

Comment:
Not a `st` user but would those patches not likely fail against a user supplied `configFile` since it has been changed?